fw/tls.c

```diff
@@ -146,8 +146,9 @@ tfw_tls_msg_process(void *conn, TfwFsmData *data)
 	r = ss_skb_process(skb, ttls_recv, tls, &tls->io_in.chunks, &parsed);
 	switch (r) {
 	default:
-		T_WARN("Unrecognized TLS receive return code -0x%X, drop packet\n",
-		       -r);
+		TFW_WITH_ADDR_FMT(&c->peer->addr, TFW_WITH_PORT, addr_str,
+			T_WARN("Session %s rejected with error -0x%X\n",
+				addr_str, -r));
```

Comment:
At this layer we already can't say enough details about the reason for the error and we can just print it as a second error message, which is not wished. I think we don't need any error messages on this layer. Please just make `ttls_recv()` to print error messages on each return path. We can get the client IP from the `TlsCtx->io_in.skb_list`.

Reply:
Addressed in https://github.com/tempesta-tech/tempesta/pull/1521/commits/f66360340665cd9dda0a6db2f140f5dcbd1a7583